app/src/main/java/de/westnordost/streetcomplete/quests/existence/CheckExistence.kt

```diff
@@ -67,7 +67,7 @@ class CheckExistence(
     override val icon = R.drawable.ic_quest_check
 
     override fun getTitle(tags: Map<String, String>): Int =
-        if (tags.containsAnyKey("name", "brand", "operator"))
+        if (tags.containsAnyKey("name", "ref", "brand", "operator"))
```

Comment:
Is it worth making this an array stored in a constant, then passing the array here and passing the array through a map and popping the first one off in getTitleArgs, then they're always in sync (although perhaps slightly less performant)?